bodhi/client/bindings.py

```diff
@@ -725,6 +738,20 @@ def update_str(self, update, minimal=False):
                 update['autokarma'], update['unstable_karma'], update['stable_karma']))
         ]
 
+        try:
+            test_status = self.get_test_status(update['alias'])
+        except (ServerError, requests.exceptions.RequestException):
+            test_status = None
+
+        if test_status:
```

Comment:
`if test_status is None` should we display that we couldn't retrieve the status ? something like `CI Status: error while requesting the tests status` ?

Reply:
Sure could do. My original thought was that I didn't want to show the ``CI Status`` line for bodhi instances not using CI and/or greenwave.
What do you think?

Reply:
I can see that we don't want to show the `CI Status` for updates that are not using CI.  
But that will lead to the case where an update using CI fails to get the tests status, this will just silently fail. This is maybe not a use case that is worth considering.

Reply:
Ok that's sounds good :+1: